styles.css

```diff
@@ -49,10 +113,53 @@ hr {
 }
 
 h1 {
+  color: grey;
   padding-top: 20px;
   padding-bottom: 20px;
 }
 
 h3 {
   padding-bottom: 10px;
+  color: white;
+}
+
+h2 {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 20px;
+}
+
+#contact-section {
+  margin-top: 20px;
+}
+
+.contact-text {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 20px;
+}
+
+.contact-list {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 30px;
+}
+
+.copyrights {
```

Comment:
- [Optional] bottom: 0; position: fixed, will stick the footer to the bottom of the page. This will avoid the footer movement.